test/models/metadata/SdesFileMetadataSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.metadata
+
+import models.{FileFormat, FileRole}
+import utils.SpecBase
+
+class SdesFileMetadataSpec extends SpecBase {
+
+  "correctly convert CashStatementFileMetadata to a Map" in new Setup {
```

Comment:
Could you please move the in clause inside a should or must like below?

"toMap" must {
   <in clause>
}